.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,41 +4,48 @@ about: Create a report about an issue or bug
 title: ''
 labels: bug
 assignees: ''
-
 ---
 
 ## Describe the bug
 <!-- A clear and concise description of what the bug is. -->
 
-## Steps to Reproduce
-<!--
+## Steps to reproduce the bug
+<!-- A step by step explanation on how to trigger the bug. -->
+
 Steps to reproduce the behavior:
+
 1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
--->
+2. Scroll down to '....'
+3. Click on '....'
 
 ## Current _broken_ behavior
-<!-- What is the current broken behaviour? -->
+<!-- What is the current broken behaviour that happens when following the steps above? -->
 
 ## Expected behavior
-<!-- A clear and concise description of what you expected to happen. -->
+<!-- A clear and concise description of what you expected to happen instead. -->
 
 ## Screenshots, videos or logs
 <!-- If applicable, add screenshots, videos or logs to help explain your problem. You can use code blocks (```) to format the logs  -->
 
 ## Tested device
- - Device:
+
+- Device:
 <!-- On which device did the bug occur? iPhone 12? Google Pixel 5? Honor View 10? -->
- - OS:
+
+- OS:
 <!-- Which OS do you use? Android 11? iOS 14? macOS 15? -->
- - App version:
-<!-- (In which version of the app does the error occur? You can find the version under Settings > "About us". Otherwise you can find a tutorial with picture under this link: https://sharezone.zendesk.com/hc/de/articles/360013355879-Welche-Version-der-App-habe-ich-installiert-) -->
+
+- App type: <!-- web app / macOS app / iOS app / Android app -->
+<!-- What app type of Sharezone did you use? The web-app in the browser? The macOS app from the AppStore? Android app? -->
+
+- App version: <!-- e.g. "1.6.1 (315)" -->
+<!-- In which version of the app does the error occur? 
+You can find the version under Settings > "About us". 
+Otherwise you can find a tutorial with pictures under this link: 
+https://sharezone.zendesk.com/hc/de/articles/360013355879-Welche-Version-der-App-habe-ich-installiert- -->
 
```

Comment:
I put spaces between the bullet points (only shows in markdown i think) so it's easier to read with all of these comments.